pkg/substitution/substitution_test.go

```diff
@@ -15,13 +15,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1_test
+package substitution_test
```

Comment:
total nitpick but i think we might be able to get away with just `substitution` now? afaik `v1alpha_test` had to be used b/c we were having some kinda like conflicts or circular dependency problems or something (might be thinking of a different issue tho - i know this happens with the builder libs a lot! - but probably wont once you're done!!)

Reply:
I think `v1alpha_test` might have been for that reason indeed. I tend to like using `_test` package as it force the test to only use what the user can use and how he will use it :stuck_out_tongue:

Reply:
ah kk! makes sense to me. ill have to noodle on it a bit and maybe ill switch to the same thing!